controllers/ixiatg_controller.go

```diff
@@ -110,8 +110,8 @@ const (
 	IXIA_C_GRPC_VERSION   string = "0.0.1-3114"
 
 	LIVENESS_DELAY   int32 = 10
-	LIVENESS_PERIOD  int32 = 1
-	LIVENESS_FAILURE int32 = 3
+	LIVENESS_PERIOD  int32 = 20
+	LIVENESS_FAILURE int32 = 10
```

Comment:
Affecting other two variables will require increasing this count (so that the net amount of time we wait before timing out is as intended)

Reply:
Failure count set to 6.